packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin.tsx

```diff
@@ -49,6 +49,11 @@ export const TooltipPlugin: React.FC<TooltipPluginProps> = ({
 
   // Add uPlot hooks to the config, or re-add when the config changed
   useLayoutEffect(() => {
+    if (plotCtx && plotCtx.plot) {
+      plotCtx.plot.root.onmouseleave = (event) => {
```

Comment:
you may instead consider binding to `.u-over`, so it disappears when leaving the grid area instead of the entire chart.

Reply:
Hmm, where can I find .u-over? I'm a bit lost here and this solution was quite hacky, any guidance appreciated.

Reply:
e.g. https://github.com/grafana/grafana/blob/cf39d5b62a0330f3e8e2ee8b5098cf82d656c473/public/app/plugins/panel/barchart/bars.ts#L177-L181

(just make sure the querySelector only runs during init and not on every redraw)

Reply:
i really should just expose this as `u.over` in future uPlot versions, even in my own demos i have to do this querySelector dance pretty often.

Reply:
Alright, I think I've cleaned this up a bit. Thanks for the help.